pkg/expr/errors.go

```diff
@@ -63,7 +63,7 @@ var DependencyError = errutil.NewBase(
 	depErrStr,
 	errutil.WithPublic(depErrStr))
 
-func makeDependencyError(refID, depRefID string) error {
+func MakeDependencyError(refID, depRefID string) error {
```

Comment:
This is the only change in @grafana/observability-metrics owned code. Just exporting this, so it can be used in tests in another package.